instance/static/html/instance/appserver.html

```diff
@@ -46,12 +46,19 @@
       </tbody>
     </table>
 
-    <button ng-disabled="is_active || !appserver.is_healthy"
-            tooltip="This will update the DNS entries so that this app server is used for the instance. Only one app server can be active at any time."
-            tooltip-placement="right"
+    <button ng-disabled="appserver.is_active || !appserver.is_healthy"
+            tooltip="This will add this app server to the list of the DNS entries for this instance."
```

Comment:
"This will add this app server to the load balancer pool"
(Generally, activating appservers should not involve DNS changes)

Same change below.